### ISSUE TITLE
http://dokan-dev.net Down

### ISSUE THREAD

**Issue by Denis2222** [apaka/dokan-net#1]
Ping ok but nomore :'(
https://code.google.com/p/win-sshfs/

**Comment by Maxhy** [comment]
Yeah dokan-dev.net is out since few weeks. The domain should have expired May 29 2015 but it was renewed the exact same day. I really don't understand why someone still pay for the domain name. Maybe it is done automatically without approval and the server behind goes down.
